src/assets/code_of_conduct.md

```diff
@@ -0,0 +1,92 @@
+<center>Windy City DevFest 2019</center>
+========================================
+<img style="display: block; margin-left: auto; margin-right: auto;" width="300" height="300" src="./event-logo.svg">
+
+<center>Code of Conduct</center>
```

Comment:
`<center>` doesn't work as intended (in github markdown), maybe try `<p style="text-align: center;">` instead.

Reply:
Github Markdown apparently strips out style :/ Have to do it in a paragraph tag 🤢